EIPS/eip-1620.md

```diff
@@ -1,296 +1,185 @@
-
-Furthermore, it is intriguing to explore the prospect of stream-specific sidechains.
-
-### Oracles
-
-The proposed specification uses block numbers to proxy time, but this need not be the only method. Albeit it would imply different trust assumptions, oracles could be used to provide a feed of timestamps. Coupled with the aforementioned idea of stream-specific sidechains, oracles could efficiently solve the problems outlined in [Block Times](#block-times).
-
-### Multi-Hop Streams
-
-Future or upgraded versions of this standard may describe "multi-hop" streams. If:
-
-1. There is a stream between A and B
-2. There is another stream between B and C
-
-There could be a way to avoid running two different streams in parallel. That is, a fraction or all of the funds being streamed from A to B could be automatically wired to C. An interesting use case for this is taxes. Instead of manually moving money around, proactively calculating how much you owe and then transfer it, a stream could atomically perform those operations for you.
+The only dangerous scenario we can think of is when [ERC-1620](https://eips.ethereum.org/EIPS/eip-1620) derived implementations end up making up a significant share of the volume of money transferred on Ethereum. It is possible, although unlikely, that some stream recipients will have an incentive to coax miners in bumping the block timestamps for them to profit. But we posit the payoff (a few seconds or minutes times the payment rate) will not be high enough for this kind of attack to be worth it.
 
 ## Implementation
 
-- [ChronosProtocol WIP implementation](https://github.com/ChronosProtocol/monorepo)
+- [Sablier](https://github.com/sablierhq/sablier/)
 
 ## Additional References
+
+- [Sablier Launch Thread](https://twitter.com/SablierHQ/status/1205533344886411264)
+- [Introducing Sablier: Continuous Salaries on Ethereum](https://medium.com/sablier/introducing-sablier-continuous-payments-on-ethereum-c2bf04446d31)
 - [Chronos Protocol Ethresear.ch Plasma Proposal](https://ethresear.ch/t/chronos-a-quirky-application-proposal-for-plasma/2928?u=paulrberg)
-- [Chronos Protocol White Paper](http://chronosprotocol.org/chronos-white-paper.pdf)
 - [Flipper: Streaming Salaries @ CryptoLife Hackathon](https://devpost.com/software/flipper-3gvl4b)
 - [SICOs or Streamed ICOs](https://ethresear.ch/t/chronos-a-quirky-application-proposal-for-plasma/2928/14?u=paulrberg)
 - [RICOs or Reversible ICOs](https://twitter.com/feindura/status/1058057076306518017)
 - [Andreas Antonopoulos' Keynote on Bitcoin, Lightning and Money Streaming](https://www.youtube.com/watch?v=gF_ZQ_eijPs)
```

Comment:
This isn't an official section, and while unofficial sections are not strictly prohibited we try to avoid them in favor of the standardized sections.  Keeping in mind that EIPs are technical specifications, and not meant as a store of history description of use cases, can we just delete this whole section?

Reply:
Marketing to products in EIPs should be removed. Having references in themselves isn't too bad, especially to avoid plagiarism.